demisto_sdk/commands/prepare_content/integration_script_unifier.py

```diff
@@ -491,12 +491,19 @@ def insert_pack_version(
                 if "// pack version:" in script_code
                 else f"// pack version: {pack_version}\n{script_code}"
             )
-        elif script_type in {".py", ".ps1"}:
+        elif script_type == ".ps1":
             return (
                 script_code
                 if "### pack version:" in script_code
                 else f"### pack version: {pack_version}\n{script_code}"
             )
+        elif script_type == ".py":
+            if "### pack version:" not in script_code:
```

Comment:
if it is already in the code, I think we should raise an error, @dantavori wdyt? 
If we don't, we may end up having multiple assignments to the env var, and only the latest will be used. 

(this also means we should remove it from the code when accepting contributions, I think we already have some code that removes some similar prefix)